packages/aws-cdk/test/integ/cli/common.bash

```diff
@@ -66,20 +66,20 @@ function prepare_fixture() {
     cp -R app/* $integ_test_dir
     cd $integ_test_dir
 
-    # if this directory is missing, but exists in any of the 
+    # if this directory is missing, but exists in any of the
     # parent directories, npm will install these packages there. lets make sure
     # we install locally.
     mkdir -p node_modules
 
     npm install \
-        @aws-cdk/core \
-        @aws-cdk/aws-sns \
-        @aws-cdk/aws-iam \
-        @aws-cdk/aws-lambda \
-        @aws-cdk/aws-ssm \
-        @aws-cdk/aws-ecr-assets \
-        @aws-cdk/aws-cloudformation \
-        @aws-cdk/aws-ec2
+        @aws-cdk/core@^1 \
```

Comment:
Why? I think this will conflict with our new 0.0.0 dev version

Reply:
I cannot get the cli integ test to work (prepare fixture) without this change. I get a npm error saying that it cannot find version 0.0.0 of those packages. Can you try?

Reply:
Damn: https://github.com/aws/aws-cdk/issues/6477